sw/host/opentitantool/Cargo.toml

```diff
@@ -18,6 +18,7 @@ env_logger = "0.8.3"
 raw_tty = "0.1.0"
 regex = "1"
 nix = "0.17.0"
+progressing = "3.0.2"
```

Comment:
Not mandatory but https://docs.rs/indicatif/0.16.2/indicatif/ is much shinier IMO

Reply:
Indeed.  I changed to using that one.